### PR TITLE
Branding Update for trento

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by stefanotorresi** [comment]
I was thinking about naming the CLI command `tn`, which is the official code for the province of Trento, while still naming the git module `trento`. Do you think we can do this here?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# SUSE Console for SAP Applications v2
+# Trento
 
-A cloud-native, web application to manage OS-related tasks for SAP Applications.
+An open cloud-native, web console for managing SAP workloads on Linux.
```
Comment (stefanotorresi) [inline README.md:3]: this conflicts with #12, maybe we could leave these diff lines out from this PR?

Comment (dirkmueller) [reply README.md]: I have rebased this now on top of #12 after merging that one.

**Comment by dirkmueller** [comment]
> I was thinking about naming the CLI command `tn`, which is the official code for the province of Trento, while still naming the Go module `trento`. Do you think we can do this here?

I'm not sure this is worth it (we don't aim for the cli command to be end user callable, it would be a container entrypoint or a systemd service or whatever else we use for launching) and "tn" already conflicts with /usr/bin/tn (telnet)

**Comment by stefanotorresi** [comment]
Good points! Let's stick with `trento` then.

**Review by stefanotorresi — APPROVED** [review]

**Review by MalloZup — COMMENTED** [review]
lgtm

**Pull request merged**